packages/connect-examples/new-example/app/components/common/ParameterInput.tsx

```
@@ -0,0 +1,511 @@
+
+  // 预设选择处理
+  const handlePresetChange = (presetTitle: string) => {
+    const preset = presets.find(p => p.title === presetTitle);
+    if (preset && preset.parameters) {
+      const newMethodParams: Record<string, unknown> = {};
+
+      // 从预设参数中获取值
+      preset.parameters.forEach((param: ParameterField) => {
+        if (param.value !== undefined && !['useEmptyPassphrase'].includes(param.name)) {
+          // 使用统一的参数处理工具
+          newMethodParams[param.name] = parseParameterValue(param.name, param.value);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)